src/Intl/Grapheme/Grapheme.php

```diff
@@ -62,7 +62,7 @@ public static function grapheme_extract($s, $size, $type = \GRAPHEME_EXTR_COUNT,
         $start = (int) $start;
 
         if (\GRAPHEME_EXTR_COUNT !== $type && \GRAPHEME_EXTR_MAXBYTES !== $type && \GRAPHEME_EXTR_MAXCHARS !== $type) {
-            if (80000 > \PHP_VERSION_ID) {
+            if (80000 >= \PHP_VERSION_ID) {
```

Comment:
I'm pretty sure the current version is correct. `80000 > \PHP_VERSION_ID` matches PHP releases before 8.0. For those releases, we return `false` because that's how `ext-mbstring` would behave there.

The `ValueError` that we throw otherwise has been introduced with PHP 8.0.0.

Reply:
After flipping the Yoda mode, you can see that `PHP_VERSION_ID > 80000` maches versions afer 8.0.0, no including 8.0.0 iself!

Reply:
Nevermind, when flipping Yoda mode, it should have to change the sign too, so it would be `PHP_VERSION_ID < 80000`

Blame Yoda for brain freeze here!

and here it's not even justified (to have Yoda mode), you can't assign values to constants with assign statement! but makes code difficult to understand if you don't do Yoda.

Reply:
making a comparison "is number x less than than 8" with `8 > x` makes my brain hurt!

Reply:
I feel you. But that's the CS that is enforced on this repo. 🤷🏻